backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc

```diff
@@ -32,8 +32,8 @@ include::modules/installing-oadp-rosa-sts.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/user-tasks#olm-installing-from-operatorhub-using-web-console_olm-installing-operators-in-namespace[Installing from OperatorHub using the web console].
-* link:https://docs.openshift.com/container-platform/4.14/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html[Backing up applications]
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/operators/user-tasks#olm-installing-from-operatorhub-using-web-console_olm-installing-operators-in-namespace[Installing from OperatorHub using the web console].
+* link:https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html[Backing up applications]
```

Comment:
xref - please

Reply:
I think we tried using an xref when we published ROSA but it didn't work which is why we had to use a hard-link instead.

Reply:
No it is a link to the OADP documentation
I have explained previously that using links to our docs is incorrect

Reply:
@anarnold97 
Thank you for your help. Your suggestion works.